draft-ietf-quic-version-negotiation.md

```diff
@@ -403,27 +404,33 @@ closing the connection with a version negotiation error. For QUIC version 1,
 version negotiation errors are signaled using a transport error of type
 VERSION_NEGOTIATION_ERROR (see {{iana-error}}).
 
-When a server receives a client's first flight, the server will first establish
-which QUIC version is in use for this connection in order to properly parse the
-first flight. For example, the server determines that QUIC version 1 is in use
+When a server receives a client's first flight, the server will first
+establish which QUIC version is in use for this connection in order to
+properly parse the first flight. This may involve examining packets
+that are not part of the handshake transcript, such as in the the
```

Comment:
```suggestion
properly parse the first flight. This may involve examining data
that is not part of the handshake transcript, such as parts of the
```